### PR TITLE
Add new committee to committee page

### PR DESCRIPTION
**Pull request by qPalee**
Need to add me and Leaf to committee page for 2023-24

### CODE REVIEW

File: _data/committee/2023-24.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-academic_year: 2023/2024
-#postfix: Elect
+Ôacademic_year: 2023/2024 #postfix: Elect
```
Comment (codemicro) [inline _data/committee/2023-24.yaml:1]: !!! funky character alert !!!

**Review by codemicro — CHANGES_REQUESTED** [review]

File: _site/_redirects
Comment (codemicro) [inline _site/_redirects:1]: Also didn't spot this again but also you probably didn't mean to delete all the redirects

**Review by codemicro — CHANGES_REQUESTED** [review]

File: _site/_redirects
Comment (qPalee) [reply _site/_redirects:1]: how have i done that

Comment (codemicro) [reply _site/_redirects:1]: absolute skillz

File: _data/committee/2023-24.yaml [hunk 2]
```diff
@@ -210,4 +210,19 @@ people:
...
+      picture: /assets/committee/2023-24/mini/lee.png
+      start-date: 31/01/2024
+      bio: >
+        Hiya! I'm Lee, your PGT Rep. My role within the committee is to
+        try and get the PGTs the engage more with the society. In my free
+        time, I enjoy playing games (mostly Destiny 2) and also watching
+        F1, darts and my football team, West Ham United. If you see me 
+        around feel free to have a chat or reach out to me on discord 
+        where I'm nearly always active :)
+
```
Comment (codemicro) [inline _data/committee/2023-24.yaml:228]: This block is indented one level too far and has ceased to be valid YAML

**Review by codemicro — CHANGES_REQUESTED** [review]

File: _data/committee/2023-24.yaml [hunk 3]
```diff
@@ -210,4 +210,19 @@ people:
       I'm looking forward to keeping that going with fun events that are accessible to the
       needs of my cohort. I may be crazy, but I'll never be scary so always feel free to
       come over for a chat! #ILoveOSM
+
  
+  - name: Lee Alabaster
+    pronouns: He/Him
+    role: Post-Graduate (Taught) Rep
+    discordtag: palee.
+    picture: /assets/committee/2023-24/mini/lee.png
```
Comment (codemicro) [inline _data/committee/2023-24.yaml:219]: Your image is a `.jpg`, not a `.png`

**Review by codemicro — CHANGES_REQUESTED** [review]

**Review by codemicro — APPROVED** [review]
👍😀👍

Not going to wait for @antikitten but he can open a new PR if he'd like

**Pull request merged**